.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: shell
+
+install: true
+
+script:
```

Comment:
Maybe run against all scripts rather than selected ones?

Reply:
I need to whitelist them anyways in the .gitignore file and since some of them don't have a `.sh` ending (which they probably should but I'm too lazy), I just use a hardcoded list.